mitosheet/src/jupyter/jupyterUtils.tsx

```diff
@@ -0,0 +1,62 @@
+import MitoAPI from "../api"
+import { notebookGetArgs, notebookOverwriteAnalysisToReplayToMitosheetCall, notebookWriteAnalysisToReplayToMitosheetCall, notebookWriteGeneratedCodeToCell } from "./notebook/pluginUtils"
+
+
+export const isInJupyterLab = (): boolean => {
+    return window.location.pathname.startsWith('/lab')
+}
+
+export const isInJupyterNotebook = (): boolean => {
+    return window.location.pathname.startsWith('/notebooks')
+}
+
+export const writeAnalysisToReplayToMitosheetCall = (analysisName: string, mitoAPI: MitoAPI) => {
+    if (isInJupyterLab()) {
+        window.commands?.execute('write-analysis-to-replay-to-mitosheet-call', {
+            analysisName: analysisName,
+            mitoAPI: mitoAPI
+        });
+    } else if (isInJupyterNotebook()) {
+        notebookWriteAnalysisToReplayToMitosheetCall(analysisName, mitoAPI);
+    }
+}
+export const overwriteAnalysisToReplayToMitosheetCall = (oldAnalysisName: string, newAnalysisName: string, mitoAPI: MitoAPI) => {
+    if (isInJupyterLab()) {
+        window.commands?.execute('overwrite-analysis-to-replay-to-mitosheet-call', {
```

Comment:
Comment and explain the order flow here in the jupterUtils. Should be a big comment at the top.